about.html

```diff
@@ -0,0 +1,176 @@
+          projects. Most of the speakers are successful in their branch.
+        </p>
+      </div>
+      <span class="contact-span">
+        Please contact us per Email for any further questions <br />
+        about Europe Global 2023 Summit! <br
+      /></span>
+      <a href="digital@europeglobalsummit.com" class="contact-href"
+        >digital@europeglobalsummit.com</a
+      >
+    </section>
+    <section class="about-award">
+      <div class="award">
+        <h3 class="certification">Europe Global 2023 Summit Logo</h3>
+        <hr />
+        <span class="logo-award"
+          >The competition took place for the first time in 2019. Creative and
+          colourful, these award winning logos reflect a perfect harmony between
+          function and design.</span
+        >
+        <img
+          class="logo-img"
+          alt="logo image"
+          src="./images/earth-globebigger.png"
+        />
+      </div>
+      <hr class="black-line" />
+    </section>
+    <section class="about-award">
+      <div class="award">
+        <h3 class="certification">See the past Digital World Summits</h3>
+        <hr />
+        <span class="logo-award"
+          >Take a look at the last two Europe Global Summits which took place in
+          Berlin</span
+        >
+      </div>
+    </section>
+    <section class="events">
+      <div class="carousel">
+        <img class="card" alt="conference" src="images/conf1.jpg" />
+        <h4 class="year">2021</h4>
+        <p class="text-event">Europe Global Summit</p>
+      </div>
+      <div class="carousel">
+        <img class="card" alt="conference" src="images/conf2.jpg" />
+        <h4 class="year">2022</h4>
+        <p class="text-event">Europe Global Summit</p>
+      </div>
+    </section>
+    <section class="partners-section" id="sponsor">
+      <h4 class="partner-text">Partners</h4>
+      <hr class="red-line" />
+      <div class="flex-img">
+        <img src="images/mozilla.png" alt="Mozzila logo" class="logo-partner" />
+        <img
+          src="images/google.png"
+          alt="Google logo"
+          class="logo-partner logo-size"
+        />
+        <img
+          src="images/naver.png"
+          alt="Naver logo"
+          class="logo-partner logo-size"
+        />
+        <img
+          src="images/daumkakao.png"
+          alt="Daumkakao logo"
+          class="logo-partner logo-size"
+        />
+        <img
+          src="images/airbnblogo.png"
+          alt="Airbnb logo"
+          class="logo-partner"
+        />
+      </div>
+    </section>
+    <footer class="footer" id="news">
+      <div class="footer-section">
+        <img
+          src="images/earth-globe.png"
+          alt="Europe Summit logo"
+          class="logo-digital"
+        />
+        <span class="footer-span">
+          2023 Geneva Switzerland
+          <br />
+          Some Rights Reserved
+        </span>
+      </div>
+    </footer>
```

Comment:
- [ ] Good job styling the footer section of your about page :+1:, but as you can see below it should have a dark background to match the template 

**Your design**

![Screenshot from 2023-02-07 08-09-34](https://user-images.githubusercontent.com/105117832/217254160-f6849c32-de7d-44b7-8d67-558e1357a259.png)

**Template**

![Screenshot from 2023-02-07 08-10-00](https://user-images.githubusercontent.com/105117832/217254202-0a0b6a00-9d1c-42e2-82be-87b798f89f0d.png)